gym_minigrid/wrappers.py

```diff
@@ -246,6 +253,78 @@ def observation(self, obs):
             'image': full_grid
         }
 
+class DictObservationSpaceWrapper(gym.core.ObservationWrapper):
+    """
+    Use a Dict Obsevation Space encoding images, missions, and directions
```

Comment:
Could you improve this docstring to explain the purpose of the wrapper.
If possible with example

Reply:
Does this look ok ? I am not sure what kind of example I could add here.

Reply:
Looks good to me